unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -401,6 +429,24 @@ private bool ItsAnOutdatedUpdate(string userId, FriendshipStatus friendshipStatu
                    && friends[userId].friendshipStatus == FriendshipStatus.FRIEND;
         }
 
+        private void AddIncomingFriendRequest(FriendRequest friendRequest)
+        {
+            UpdateFriendshipStatus(new FriendshipUpdateStatusMessage
+                { action = FriendshipAction.REQUESTED_FROM, userId = friendRequest.From });
+
+            TotalReceivedFriendRequestCount += 1;
+            OnTotalFriendRequestUpdated?.Invoke(TotalReceivedFriendRequestCount, TotalSentFriendRequestCount);
+        }
+
+        private void AddOutgoingFriendRequest(FriendRequest friendRequest)
+        {
+            UpdateFriendshipStatus(new FriendshipUpdateStatusMessage
+                { action = FriendshipAction.REQUESTED_TO, userId = friendRequest.To });
+
+            TotalSentFriendRequestCount += 1;
```

Comment:
why does the controller has to handle the friend request count and not the bridge itself? this responsibility was handled by kernel before (WebInterfaceBridge), and now we have retro-compatibility issues if we have to keep supporting it by a feature flag